packages/ui-vue/src/components/AppButton.vue

```diff
@@ -4,24 +4,19 @@
   defineOptions({
     inheritAttrs: false,
   });
-  defineProps({
```

Comment:
I didn't realize this [already existed](https://primevue.org/button/#api.button.props.size) so I removed it.